vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTrip.java

```diff
@@ -0,0 +1,138 @@
+package org.egov.vehicle.trip.web.model;
+
+import java.util.List;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import org.egov.common.contract.request.User;
+import org.egov.vehicle.web.model.AuditDetails;
+import org.egov.vehicle.web.model.Vehicle;
+import org.springframework.validation.annotation.Validated;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * Request schema of VehicleTrip.  
+ */
+@Validated
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2020-12-23T12:08:13.326Z[GMT]")
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class VehicleTrip   {
+
+  @JsonProperty("id")
+  private String id = null;
+
+  @NotNull
+  @NotBlank
+  @JsonProperty("tenantId")
+  private String tenantId = null;
+  
+  @JsonProperty("tripOwner")
+  private User tripOwner = null;
+  
+  @JsonProperty("tripOwnerId")
+  private String tripOwnerId = null;
+  
+  @JsonProperty("driver")
+  private User driver = null;
+  
+  @JsonProperty("driverId")
+  private String driverId = null;
+  
+
+  @NotNull
+  @JsonProperty("vehicle")
+  private Vehicle vehicle;
+  
+
+  @JsonProperty("vehicleId")

```

Comment:
add @size on all string fields and @Valid on all Class objects in all models

Reply:
Done